example_projects/transfer/nlb_observability_stack/main.tf

```diff
@@ -0,0 +1,26 @@
+terraform {
+  required_providers {
+    doublecloud = {
+      source = "registry.terraform.io/doublecloud/doublecloud"
+    }
+    aws = {
+      source  = "hashicorp/aws"
+      version = "= 5.19.0"
+    }
+  }
+}
+
+provider "doublecloud" {
+  authorized_key = file("authorized_key.json")
+  endpoint       = var.service_account_endpoint
```

Comment:
do not use `endpoint` and `token_url` parameters by default

Reply:
changed to conditional use of edpoint & token_url